src/pages/private/dashboard/index.js

```diff
@@ -1,11 +1,53 @@
-import React from "react";
+import React, { useState } from "react";
 import Button from "@mui/material/Button";
+import AppDispatcher from "@redux/dispatchers/appDispatcher";
+import Grid from "@mui/material/Grid";
+import { API, NetworkManager } from "@network/core";
 
 function Dashboard() {
+  const [api, setApi] = useState({
+    loading: true,
+    gallery: []
+  });
+  const networkObj = new NetworkManager(API.PRIVATE.IMAGE_GALLERY);
+  networkObj
+    .httpRequest(true)
+    .then((res) => res.data)
+    .then((item) => {
```

Comment:
no need of chaining the .then here. You should have got the data in the first then function because an equivalent promise will be exposed to the subsequent then in the method chain.

Also item is a list of photo so you should rename it to either data or photos. `item` is not correct variable naming

Reply:
resolved chaining .then